src/components/Speedometer.tsx

```diff
@@ -9,6 +9,11 @@ interface IProps {}
 interface IState {
     gw2data_ipc: IGw2MumbleLinkData | null;
     gw2data_ws: IGw2MumbleLinkData | null;
+    now_position: any;
+    now_timestamp: any;
+    last_position: any;
+    last_timestamp: any;
+    speed: any;

```

Comment:
speed is number :) Also consider if it should be null before calculating speed the first time.